src/routes/open/container/Open.jsx

```diff
@@ -21,6 +21,20 @@ type State = {
   safeTx: string,
 }
 
+const createSafe = async (safeContract, values, userAccount, addSafe) => {
```

Comment:
Extracted this function outside the component, just in case the component is unmounted (because the user leaves the current view) the transaction is finished and the safe added  to localstorage